specification/orbital/resource-manager/Microsoft.Orbital/stable/2022-11-01/orbital.json

```diff
@@ -0,0 +1,2223 @@
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Name of the ground station."
+        },
+        "location": {
+          "type": "string",
+          "description": "Azure region."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource type."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "allOf": [
+            {
+              "$ref": "#/definitions/AvailableGroundStationProperties"
+            }
+          ],
+          "description": "The properties bag for this resource."
+        }
+      },
+      "description": "Ground Stations available to schedule Contacts."
+    },
+    "AvailableGroundStationProperties": {
+      "type": "object",
+      "properties": {
+        "city": {
+          "type": "string",
+          "description": "City of ground station."
+        },
+        "providerName": {
+          "type": "string",
+          "description": "Ground station provider name."
+        },
+        "longitudeDegrees": {
+          "type": "number",
+          "description": "Longitude of the ground station in decimal degrees."
+        },
+        "latitudeDegrees": {
+          "type": "number",
+          "description": "Latitude of the ground station in decimal degrees."
+        },
+        "altitudeMeters": {
+          "type": "number",
+          "description": "Altitude of the ground station."
+        },
+        "releaseMode": {
+          "type": "string",
+          "enum": [
+            "Preview",
+            "GA"
+          ],
+          "x-ms-enum": {
+            "name": "releaseMode",
+            "modelAsString": true
+          },
+          "description": "Release Status of a ground station."
+        }
+      },
+      "description": "Properties object for Available groundstation."
+    },
+    "Capability": {
+      "type": "string",
+      "description": "Capability of the Ground Station.",
+      "enum": [
+        "EarthObservation",
+        "Communication"
+      ],
+      "x-ms-enum": {
+        "name": "Capability",
+        "modelAsString": true
+      }
+    },
+    "ResourceIdListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "id": {
+                "type": "string",
+                "description": "The Azure Resource ID."
+              }
+            }
+          },
+          "description": "A list of Azure Resource IDs."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URL to get the next set of results.",
+          "readOnly": true
+        }
+      },
+      "description": "Response for an API service call that lists the resource IDs of resources associated with another resource."
+    },
+    "Etag": {
```

Comment:
>Etag [](http://example.com/codeflow?start=5&length=4)

Looks like Etag is not being used anywhere? If that's the case, please remove the definition.